styles.css

```diff
@@ -1001,3 +987,444 @@ h1 {
+    padding: 0;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    position: absolute;
+    width: 255px;
+    height: 40px;
+    right: 154px;
+    top: calc(40% - 40px / 2);
+    color: white;
+    gap: 10px;
+  }
+
+  .headline {
+    position: absolute;
+    width: 100%;
+    height: 870px;
+    left: 0;
+    top: 72px;
+    background: #1c1a19;
+    background-image: url(Images/bg-d.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: 553px 769.27px;
+  }
+
+  .headline-2 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 10px;
+    position: absolute;
+    width: 566px;
+    height: 739px;
+    left: 143px;
+    top: 52px;
+  }
+
+  h1 {
+    width: 566px;
+    height: 160px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .link {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    width: 20px;
+    height: 98.96px;
+    left: 36px;
+    top: 234px;
+    gap: 12px;
+  }
+
+  .supporting-text {
+    width: 566px;
+    height: 24px;
+    display: flex;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .description {
+    width: 566px;
+    height: 24px;
+    display: flex;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .recent-works {
+    width: 100%;
+    height: 1805px;
+    top: 870px;
+  }
+
+  .title {
+    display: flex;
+    flex-direction: row;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .head-title {
+    display: inline-flex;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    width: 315px;
+    height: 52px;
+    padding: 0 10px 0 0;
+    margin-left: 100px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    border: 1px solid #212121;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    top: 157px;
+    align-self: center;
+  }
+
+  .sub-line {
+    display: none;
+  }
+
+  .featured-image {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    position: absolute;
+    width: 1156px;
+    height: 448px;
+    left: -153px;
+    top: 158px;
+  }
+
+  .img {
+    width: 484px;
+    height: 448px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    background-image: url(Images/Img-Placeholder.png);
+    background-repeat: no-repeat;
+    margin: 34px 0 0 164px;
+    background-size: 400px 380px;
+  }
+
+  .right-block {
+    width: 448px;
+    height: 348px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .title-post {
+    width: 448px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-top: 55px;
+  }
+
+  .sup {
+    width: 448px;
+    height: 100px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .tags {
+    margin-top: 0;
+  }
+

```

Comment:
- [ ] Kindly fix your hover style on the works cards. When you a hover over each one, a button to `see project` should be displayed

![image](https://user-images.githubusercontent.com/93630700/208180054-bf33e0b6-88b4-4b3f-917f-64c6e22053f9.png)